lib/va_profile/v3/address_validation/address_suggestions_response.rb

```diff
@@ -23,7 +24,8 @@ def initialize(candidate_res)
                 ).to_h
               }
             end,
-            override_validation_key:
+            override_validation_key:,
+            validation_key:
```

Comment:
So we have to return this same key twice but with two different names?

Reply:
@rmtolmach ContactInfoV1 required validation_key, where as ContactInfoV2 requires override_validation_key. The validation_key param is heavily referenced in the front end and it's best to ensure the VAProfile integration is working before modifying the front end.

i'm not for sure if this will work yet. I cannot test this locally because the response is mocked in local and dev env.